src/packages/src/xroad/redhat/SPECS/common.inc

```diff
@@ -25,3 +25,7 @@ if [ "$1" -gt 1 ] ; then \
     fi \
 fi \
 %{nil}
+
+%define systemd_try_restart() \
+systemctl --quiet try-restart %{?*} >/dev/null 2>&1 || : \
+%{nil}
```

Comment:
Ha!  this ':' is so clever shortcut for writing 'true' that I'm really impressed!  But for sake of the clarity, it could be more descriptive to use 'true', though.

Reply:
Using `|| :` to ignore error return is a common (ba)sh idiom.